crates/sui-bridge/src/types.rs

```diff
@@ -778,6 +817,47 @@ mod tests {
         assert_eq!(bytes, Hex::decode("5355495f4252494447455f4d4553534147450101000000000000005e0b010268b43fd906c0b8f024a18c56e06744f7c6157c65acaef39832cb995c4e049437a3e2ec6a7bad1ab5").unwrap());
     }
 
+    #[test]
+    fn test_emergency_action_encoding() {
+        let blocklist_action = BridgeAction::EmergencyAction(EmergencyAction {
```

Comment:
nit re naming: this and the action on line 841 are not blocklist actions. They are pause and unpause actions.